source/fundamentals/bson.txt

```diff
@@ -226,6 +226,8 @@ This example performs the following actions:
     to use the ``"json"`` struct tag if a ``"bson"`` struct tag is not specified
   - Sets the ``NilSliceAsEmpty`` field to ``true``, which instructs the driver
     to marshal ``nil`` Go slices as empty BSON arrays
+  - Sets the ``OmitEmpty`` field to ``true``, which instructs the driver to globally omit
+    empty values from the marshaled BSON, rather than set empty values to ``null`` 
```

Comment:
S: Add an entry in whats new for this feature

Reply:
Done!